ecs-init/config/common.go

```diff
@@ -203,3 +203,13 @@ func AgentDockerLogDriverConfiguration() godocker.LogConfig {
 		},
 	}
 }
+
+// InstanceConfigDirectory returns the location on disk for custom instance configuration
+func InstanceConfigDirectory() string {
+	return directoryPrefix + "/usr/lib/ecs"
```

Comment:
I've given this some thought, chatted with @jhaynes a bit, and I think this should be `/var/lib/ecs`. 

Ideally, the packaging should be able to provide some "vendor" defaults that are included in the `ecs-init` package. Settings for baked or pre-configured AMIs would reside in `/var/lib/ecs`; user configured settings would reside in `/etc/ecs`. This particular change is targeting the case of pre-configuration, rather than vendor defaults, from what I can tell so I suggest we use `/var/lib/ecs`.

This layout would have us loading configuration in this order:

1st: `/usr/share/ecs`, the package's defaults
2nd: `/var/lib/ecs`, the system's baked in configuration (overloadable for use in the case of baked AMIs)
3rd: `/etc/ecs`, the user's general configuration

And though we aren't vending an `ecs-init` package today with configuration in `/usr/share/ecs`, we may want to consider doing so in order to provide the package a means to suggest appropriate defaults.

Reply:
sounds good to me, I can change this to `/var/lib/ecs`